Core/Src/segment.c

```diff
@@ -5,8 +5,55 @@
 #include <stdlib.h>
 #include <string.h>
 
+// TEMPORARY
+
+#include "common.h"
+#include "adBms6830CmdList.h"
+#include "adBms6830GenericType.h"
+#include "serialPrintResult.h"
+#include "adBms6830ParseCreate.h"
+#include "mcuWrapper.h"
+
+#define ALL_GPIOS_ARE_INPUTS 0x3FF
+
+typedef enum {
+	DISCHARGE_ENABLED = 0,
+	MUTE_ACTIVATED_DISCHARGE_DISABLED = 1
+} MUTE_ST;
+
+#define TOTAL_IC 1
+cell_asic IC[TOTAL_IC];
+
+RD REDUNDANT_MEASUREMENT = RD_OFF;
+CH AUX_CH_TO_CONVERT = AUX_ALL;
+CONT CONTINUOUS_MEASUREMENT = SINGLE;
+OW_C_S CELL_OPEN_WIRE_DETECTION = OW_OFF_ALL_CH;
+OW_AUX AUX_OPEN_WIRE_DETECTION = AUX_OW_OFF;
+PUP OPEN_WIRE_CURRENT_SOURCE = PUP_DOWN;
+DCP DISCHARGE_PERMITTED = DCP_OFF;
+RSTF RESET_FILTER = RSTF_OFF;
+ERR INJECT_ERR_SPI_READ = WITHOUT_ERR;
+
+/*Loop Measurement Setup These Variables are ENABLED or DISABLED Remember ALL CAPS*/
+LOOP_MEASURMENT MEASURE_CELL =
+	ENABLED; /*   This is ENABLED or DISABLED       */
+LOOP_MEASURMENT MEASURE_AVG_CELL =
+	ENABLED; /*   This is ENABLED or DISABLED       */
+LOOP_MEASURMENT MEASURE_F_CELL =
+	ENABLED; /*   This is ENABLED or DISABLED       */
+LOOP_MEASURMENT MEASURE_S_VOLTAGE =
+	DISABLED; /*   This is ENABLED or DISABLED       */
+LOOP_MEASURMENT MEASURE_AUX =
+	DISABLED; /*   This is ENABLED or DISABLED       */
+LOOP_MEASURMENT MEASURE_RAUX =
+	DISABLED; /*   This is ENABLED or DISABLED       */
+LOOP_MEASURMENT MEASURE_STAT =
+	ENABLED; /*   This is ENABLED or DISABLED       */
```

Comment:
isnt this block kinda obsolete?

Reply:
leaving in for debugging will be gone later